src/sherpa_ai/connectors/chroma_vector_store.py

```diff
@@ -0,0 +1,78 @@
+import uuid
+
+import chromadb
+from chromadb.utils import embedding_functions
+from langchain.docstore.document import Document
+from langchain.text_splitter import CharacterTextSplitter
+
+import sherpa_ai.config as cfg
+
+
+class ChromaVectorStore:
```

Comment:
Since this will likely be used by users. I  think we should write more docstring for it so it will be included in API docs.